wallet/udb/addressdb.go

```diff
@@ -78,6 +79,15 @@ type dbBIP0044AccountRow struct {
 	name                      string
 }
 
+// dbVotingAccountRow houses additional information stored about a
+// voting account in the database.
+type dbVotingAccountRow struct {
+	dbAccountRow
+	pubKeyEncrypted []byte
+	lastUsedIndex   uint32
```

Comment:
internal or external?

Reply:
This is also missing the last returned index, which is needed to respect the gap limit